### DISCUSSION TITLE
Temp shown as -273

### DISCUSSION THREAD

**Discussion by Tipo874** [mp-se/gravitymon#81] [Q&A] [answered]
I flashed a new iSpindel with 1.0.0 and added it to my fermenter yesterday.
It started out fine, but after a few hours it stopped logging to brewfather.
I pulled the iSpindel out and noticed a tiny drop of wort inside the tube, not enough to reach any electronics, as far as I can see.
I dried it out and brushed it with alcohol and also rinsed the tube with alcohol.
Today I noticed that it no longer logs the temperature, and when I check, it reports the temp as -273. I assume it either reports 0 Kelvin, or that something causes the temp to be reported as 0 K. The gyros seems to work OK, which puzzles me as I understand that the temp sensor it uses as built into the gyro.

Any idea if this is a HW issue or a SW problem?

**Comment by mp-se** [comment]
Check the q&a in the docs, hardware issue

**Reply by Tipo874** [reply]
Ok, but I was under the impression that the DS18B20 wasn't used with this firmware and that the temp sensor in the gyro is used instead, is this not so?
Nm, I found the setting for using the gyro temp instead, which appears to work.

**Reply by mp-se** [reply]
Not by default, you need to activate that feature under hw settings

**Comment by Tipo874** [comment, marked as answer]
I noticed some crust around the pins for the DS18B20, and some other places. After cleaning that up both sensors now work :)
Edit: Turns out I could not switch back to the DS18B20, not sure if this is a bug or not. After re-flashing I see that the temp is still reported as -273. Will use the gyro sensor for now.
